moped-editor/src/views/projects/projectView/ProjectComponents/utils/map.js

```diff
@@ -101,7 +101,26 @@ export const zoomMapToFeatureCollection = (
   if (!mapRef?.current) return;
 
   const bboxOfFeatureCollection = bbox(featureCollection);
-  mapRef.current.fitBounds(bboxOfFeatureCollection, fitBoundsOptions);
+
+  /**
+   * There is a known issue with fitBounds throwing an error with padding on smaller screens
+   * So, we try to fitBounds with padding, and if that fails, we try again without padding
+   * @see https://stackoverflow.com/a/76011418
+   * */
+  try {
+    return mapRef.current.fitBounds(bboxOfFeatureCollection, fitBoundsOptions);
+  } catch {
+    console.log("fitBounds failed, trying again without padding", {
+      fitBoundsOptions,
+      screenWidth: window.innerWidth,
+    });
+  }
+
+  try {
+    mapRef.current.fitBounds(bboxOfFeatureCollection);
+  } catch {
+    console.log("Unable to fitBounds to feature collection");
```

Comment:
since this would be unusual, might be good to include the bboxFc or the entire `featureCollection` in the log message

Reply:
Good call - I'll add the feature collection 🙏